src/Channel.cpp

```diff
@@ -144,6 +144,14 @@ bool	Channel::isInvited(const Peer &peer) const
 	return std::find(this->_invitations.begin(), this->_invitations.end(), peer.getFd()) != this->_invitations.end();
 }
 
+void	Channel::removeInvitation(const Peer &peer)
+{
+	std::vector<int>::iterator it;
+	it = std::find(this->_invitations.begin(), this->_invitations.end(), peer.getFd());
+	if (it != this->_invitations.end())
```

Comment:
No need to check, `std::map::erase` do it already :)

Reply:
bad request :-1: